src/Microsoft.Data.Analysis/ArrowStringDataFrameColumn.cs

```diff
@@ -118,11 +138,11 @@ private void SetValidityBit(long index, bool value)
         }
 
         /// <summary>
-        /// Returns an enumerable of immutable buffers representing the underlying values in the Apache Arrow format
+        /// Returns an enumeration of immutable buffers representing the underlying values in the Apache Arrow format
         /// </summary>
-        /// <remarks>Null values are encoded in the buffers returned by GetReadOnlyNullBitmapBuffers in the Apache Arrow format</remarks>
+        /// <remarks><see langword="null" /> values are encoded in the buffers returned by GetReadOnlyNullBitmapBuffers in the Apache Arrow format</remarks>
         /// <remarks>The offsets buffers returned by GetReadOnlyOffsetBuffers can be used to delineate each value</remarks>
-        /// <returns>IEnumerable<ReadOnlyMemory<byte>></returns>
+        /// <returns>An IEnumerable<see cref="ReadOnlyMemory{Byte}"/> whose elements are the raw data buffers for the UTF8 string values.</returns>
```

Comment:
This is malformed.

It should be:

```
/// <returns>An <see cref="IEnumerable{ReadOnlyMemory{Byte}}"/> whose elements are the raw data buffers for the UTF8 string values.</returns>
```

Reply:
Repeated below.

Reply:
I tried this first. VS reports that the cref tag is syntactically incorrect :/. Weird. Maybe I'll stick with an enumeration of for now and we can deal with it later.